content/en/docs/components/pipelines/operator-guides/installation/_index.md

```diff
@@ -32,11 +32,39 @@ You should be familiar with [Kubernetes](https://kubernetes.io/docs/home/),
 >
 > This workaround was verified on Minikube using `v2.0.0`, and aligns with community suggestions in [kubeflow/pipelines#9546](https://github.com/kubeflow/pipelines/issues/9546). It may also help users facing similar pod crash issues in other environments or newer versions.
 
+## Deploying Kubeflow Pipelines in Kubernetes Native API Mode
+
+Kubeflow Pipelines can be deployed in Kubernetes native API mode, which stores pipeline definitions as Kubernetes Custom Resources instead of using external DB storage. This mode provides better integration with Kubernetes native tooling and GitOps workflows.
+
+**⚠️ Version Requirement**: Kubernetes native mode is available starting from Kubeflow Pipelines version 2.14.0.
+
+**⚠️ Prerequisites**: cert-manager is required for this deployment as it provides certificates for the admission webhooks used in pipeline validation.
+
+1. Deploy the Kubeflow Pipelines in Kubernetes Native Mode:
+
+     ```bash
+     export PIPELINE_VERSION={{% pipelines/latest-version %}}
+     kubectl apply -k "github.com/kubeflow/pipelines/manifests/kustomize/cluster-scoped-resources?ref=$PIPELINE_VERSION"
+     kubectl wait --for condition=established --timeout=60s crd/applications.app.k8s.io
+     kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.18.2/cert-manager.yaml
+     kubectl wait --for=condition=Ready pod -l app.kubernetes.io/instance=cert-manager -n cert-manager --timeout=300s
+     kubectl apply -k "github.com/kubeflow/pipelines/manifests/kustomize/env/cert-manager/platform-agnostic-k8s-native?ref=$PIPELINE_VERSION"
+     ```
+
+> 💡 **Alternative**: For multi-user environments with multiple teams or users requiring isolation and RBAC controls on who can access what pipelines, you can use the multi-user Kubernetes native mode:
```

Comment:
Can we just make it the default in kubeflow platform/manifests or is it not tested enough compared to platform-agnostic-multi-user @HumairAK